### ISSUE TITLE
Task and Project controls are not accessible with keyboard:A11y_.NET maui_User can create a new task_Keyboard

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#30891]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #WCAG2.1.1; #Keyboard; #BM_.NETMaui_Mac_JUL2025; #DesktopApp; #Device; #FTP; #A11ySev1; #.NET MAUI SDKs; #Mac;

## Environment Details:
Application: Developer Balance (.NET Maui)
MAC
M2

## Repro Steps:

1. Install and open the Developer Balance app
2. TAB till Add task button and press ENTER key.
3. TAB till Task and Project controls
4. Observe the issue 

## Actual Result:
When we navigate with TAB key, Task and Project controls are not accessible with keyboard
**Note:** 
For create a project screen Name, Description, Category controls and Tags are not accessible with keyboard

## Expected Result:
All the control create a new task and Creta a new project screen should be accessible with keyboard. 

## User Impact:
Keyboard only user will unbale to access controls present in create a new task and Creta a new project screen.

## Attachment

https://github.com/user-attachments/assets/5a951ea9-49dc-48d5-aeb2-e37a4bf28d7c

Similar issue attachment of create new project screen

https://github.com/user-attachments/assets/7ad46cab-4ff8-4ae2-8290-5cc9cd5ac32c

**Comment by ghost** [comment]
This issue has been verified in Visual Studio Code 1.101.0 with MAUI versions 9.0.0 and 9.0.90. It can be reproduced on MacCatalyst platform.

**Comment by jsuarezruiz** [comment]
@sheiksyedm I noticed this has been reopened, could you share the current status? Let me know if there’s any way I can help.

**Comment by kapilvaishna** [comment]
Issue is repro in version 8.0 (8)

https://github.com/user-attachments/assets/1646113f-71aa-478f-b0ce-cb91bc09b4b1

**Comment by SuthiYuvaraj** [comment]
@kapilvaishna, I have verified the Tab Navigation in the net10 DeveloperBalance app, and it works fine. The related fixes have already been moved. For reference, I have also attached a video demonstrating the behavior

https://github.com/user-attachments/assets/ac8b9a2b-4722-4d38-a05f-de49ceceec53

**Comment by kapilvaishna** [comment]
@SuthiYuvaraj Issue is fixed partially now Name, Description, Task edit field are accessible with  keyboard but for Project and Category combo boxes list items are still not accessible with keyboard and Tags are also not accessible with Keyboard.

Developer balance version 10 (10)
Chip Apple M2

https://github.com/user-attachments/assets/48d580ff-c06d-48de-9bc2-00c403b7d41d

**Comment by kubaflo** [comment]
Hi @kapilvaishna! what's the status of this task. The fix has been merged :)

**Comment by kapilvaishna** [comment]
@PureWeen @kubaflo Now only issue is list item of Project and Category combo box are not accessible with keyboard if needs separate bug for this issue let me know I we will close this bug  and log a new bug.

https://github.com/user-attachments/assets/48ec742f-78e2-4730-b26c-b0687b726eb3

**Comment by SuthiYuvaraj** [comment]
Hi @kapilvaishna, I am unable to reproduce the issue on my Mac machine using .NET 11 with Accessibility → Full Keyboard Access enabled. Please refer to the video below for reference.

https://github.com/user-attachments/assets/d555a5f2-42e0-473f-8b2e-55984c25a2c1
